### ISSUE TITLE
Upload test URL not found

### ISSUE THREAD

**Issue by frink182** [code-inflation/cfspeedtest#100]
The upload URL seems to have changed recently, now I'm seeing this:

```
Download 100KB  [==============================]  746.41 mbit/s | 100KB in    1ms -> status: 200 OK
Download 1MB    [==============================]  526.52 mbit/s |   1MB in   15ms -> status: 200 OK
Download 10MB   [==============================]  500.77 mbit/s |  10MB in  159ms -> status: 200 OK
Download 25MB   [==============================]  308.70 mbit/s |  25MB in  647ms -> status: 200 OK
Upload 100KB    [==============================]   12.94 mbit/s | 100KB in   61ms -> status: 404 Not Found
Upload 1MB      [==============================]   97.78 mbit/s |   1MB in   81ms -> status: 404 Not Found
Upload 10MB     [==============================]  744.52 mbit/s |  10MB in  107ms -> status: 404 Not Found
Upload 25MB     [==============================]  1538.91 mbit/s |  25MB in  129ms -> status: 404 Not Found
```

**Comment by code-inflation** [comment]
It seems the upload endpoint only works with https now

**Comment by code-inflation** [comment]
Fixed in latest release (version `1.2.4` https://github.com/code-inflation/cfspeedtest/releases/tag/v1.2.4)

```
Download 100KB  [==============================]  448.47 mbit/s | 100KB in    1ms -> status: 200 OK
Download 1MB    [==============================]  424.44 mbit/s |   1MB in   18ms -> status: 200 OK
Download 10MB   [==============================]  466.42 mbit/s |  10MB in  171ms -> status: 200 OK
Download 25MB   [==============================]  322.78 mbit/s |  25MB in  619ms -> status: 200 OK
Upload 100KB    [==============================]   23.30 mbit/s | 100KB in   34ms -> status: 200 OK
Upload 1MB      [==============================]   71.31 mbit/s |   1MB in  112ms -> status: 200 OK
Upload 10MB     [==============================]   93.47 mbit/s |  10MB in  855ms -> status: 200 OK
```
